packages/lib/CalEventParser.ts

```diff
@@ -150,7 +150,7 @@ export const getUid = (calEvent: CalendarEvent): string => {
 };
 
 const getSeatReferenceId = (calEvent: CalendarEvent): string => {
-  return calEvent.attendeeSeatId ? `seatReferenceUid=${calEvent.attendeeSeatId}` : "";
+  return calEvent.attendeeSeatId ? calEvent.attendeeSeatId : "";
```

Comment:
This was returning a query param for both scenarios and one of them wasn't required so the result was a malformed url.